backend/qemu.pm

```diff
@@ -540,7 +544,10 @@ sub start_qemu {
             }
         }
 
-        if ($vars->{BIOS}) {
+	if ($vars->{UEFI_PFLASH}) {
+            runcmd('cp', $vars->{BIOS}, 'ovmf.bin');
```

Comment:
I'm not a huge fan of calling external tools for this - File::Copy offers a fine copy function